command/helper/config.go

```diff
@@ -23,6 +23,7 @@ type Config struct {
 	BlockGasTarget string                 `json:"block_gas_target"`
 	GRPCAddr       string                 `json:"rpc_addr"`
 	JSONRPCAddr    string                 `json:"jsonrpc_addr"`
+	Telemetry      *Telemetry             `json:"telemetrics"`
```

Comment:
I think this should be `telemetry`, not `telemetrics`, as far as I understand from the docs: https://github.com/0xPolygon/polygon-sdk-docs/pull/25/files